scripts/release-templates.js

```diff
@@ -71,13 +71,11 @@ async function build() {
         appId: 'latency',
         apiKey: '6be0576ff61c053d5f9a3225e2a90f76',
         indexName:
-          libraryName === 'autocomplete.js'
+          libraryName === 'autocomplete'
```

Comment:
Technically a user could manually have written a template, making this a breaking change, but I don't think it's a worry

Reply:
You're right, we don't need this change anyway.